### DISCUSSION TITLE
Capitalization recovery without punctuation

### DISCUSSION THREAD

**Discussion by Robert-K-S** [NVIDIA-NeMo/NeMo#5438] [Q&A]
Is the NeMo PunctuationCapitalizationModel capable of producing an output that doesn't muck with punctuation, or would I have to write a new CapitalizationModel?

**Comment by ekmb** [comment]
There's no flag to turn punctuation off in pre-trained models. You can modify the model's inference method and comment out this line https://github.com/NVIDIA/NeMo/blob/46d923c48cbf27f744772ceaf2b20e49c73941bb/nemo/collections/nlp/models/token_classification/punctuation_capitalization_model.py#L1070
